ToDoWebApp/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ToDoWebApp.Data.Intefaces;
+using ToDoWebApp.Models;
+
+namespace ToDoWebApp.Controllers
+{
+    public class CategoryController : Controller
+    {
+
+        public static List<Category> asList;
+        public static bool pereita;
+
+        public CategoryController(ICategoryRepository IcategoryRepository)
+        {
+            if (pereita.Equals(false))
```

Comment:
Redundant boolean comparison. Variable `pereita` itself is bool statement could be simplified to:
`if(!pereita)`